application/console/config/main.php

```diff
@@ -2,23 +2,8 @@
 return [
     'id' => 'app-console',
     'basePath' => dirname(__DIR__),
-    'bootstrap' => ['log', 'gii'],
-    'controllerNamespace' => 'console\controllers',
```

Comment:
`log` should be bootstrapped in common and I think you need `gii` in console for rebuilding base models. Pretty sure we need `controllerNamespace` for a `CronController` to be called from CLI.